pkg/autoops/api/error.go

```diff
@@ -151,6 +151,10 @@ var (
 		codes.FailedPrecondition,
 		"autoops progressive rollout: can not use progressive rollout when feature has targets",
 	)
+	statusProgressiveRolloutFeatureRuleExists = gstatus.New(
+		codes.FailedPrecondition,
+		"autoops progressive rollout: can not use progressive rollout when feature has rules",
```

Comment:
nit: `autoops: can not create a progressive rollout when the feature flag has targeting rules set`